modules/ROOT/pages/_partials/include-manage-proxies.adoc

```diff
@@ -51,7 +51,7 @@ You can configure HTTP proxies so that outbound components can communicate with
 [NOTE]
 Starting Runtime Fabric version 2.6.22, the following HTTP proxy is needed for Anypoint Monitoring sidecar or agents as well. SOCK5 proxy is not required for Mule apps deployed on Runtime Fabrics with version 2.6.22 onwards
 
-Updating an HTTP proxy causes each machine in the cluster to restart all pods running on it. Although no downtime occurs if you are running multiple replicas, you should perform HTTP proxy updates during a maintenance window. 
+The rtf-agent pod and any Mule app that needs the proxy applied requires to be restarted. Although no downtime occurs if you are running multiple replicas, you should perform HTTP proxy updates during a maintenance window. 
```

Comment:
```suggestion
You must restart the `rtf-agent` pod and any Mule app that requires a proxy. Although no downtime occurs if you are running multiple replicas, you should perform HTTP proxy updates during a maintenance window. 
```

Do people know what the rtf-proxy is? I don't see that or "RTF proxy" or "Runtime Fabric proxy" mentioned anywhere else in the docs. I see one mention of a "proxy configuration from Runtime Fabric".